go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -308,6 +309,32 @@ func (m *ModelClient) create(opModel *OperationModel) ([]ovsdb.Operation, error)
 			return nil, err
 		}
 		ops = append(ops, waitOps...)
+	} else if o[0].Table == "Logical_Router_Policy" {
+		lrp, ok := opModel.Model.(*nbdb.LogicalRouterPolicy)
+		if !ok {
+			return nil, fmt.Errorf("expected LogicalRouterPolicy model, got: %v", opModel.Model)
```

Comment:
No sure we get anything useful from `%v", opModel.Model`
Perhaps try `%v", reflect.ValueOf(opModel.Model).Type().String()`

Reply:
fixed

Reply:
ooops `%s", reflect.ValueOf(opModel.Model).Type().String()`

sorry !!